### ISSUE TITLE
Add TEST target to externpro solution

### ISSUE THREAD

**Issue by stowawayi** [smanders/externpro#128]
Add TEST target to recursively run all tests defined within each library

**Comment by smanders** [comment]
similar request... run <project> unit test as part of [in|ex]ternpro build (example: Sdvideo from internpro)

**Comment by smanders** [comment]
smanders/externpro will be archived with no further changes to the repo (I'll copy issues that still are relevant to https://github.com/externpro/externpro/issues)

smanders/externpro was renamed and relocated to https://github.com/externpro/exdlpro -- but exdlpro is simply a stop-gap repository... projects are being removed in favor of each project becoming a standalone build that uses [externpro/externpro](https://github.com/externpro/externpro) as a submodule and creates their own devel packages (a .tar.xz file for each supported platform that contains all the development artifacts: lib, headers, etc necessary to build against)

as I've converted projects to devel packages, I've made sure these projects build and run their unit tests wherever possible (in the same workflow that creates the devel package) using CMakePresets... for example on Linux https://github.com/externpro/externpro/blob/25.06/cmake/presets/xpLinuxNinja.json#L61-L90

to build an externpro devel package:
```
cmake --preset=Linux
cmake --workflow --preset=Linux
```
and you can see that the `workflow` includes running tests on Release and Debug https://github.com/externpro/externpro/blob/25.06/cmake/presets/xpLinuxNinja.json#L134-L141

there are also tests that run as part of buildpro... https://github.com/externpro/buildpro/blob/25.06/test/CMakeLists.txt -- any time I add another supported externpro devel package, I update buildpro with a test for that project to validate that the devel package is usable on all supported platforms https://github.com/externpro/buildpro/blob/25.06/test/CMakeLists.txt

long story short: this issue is obsolete and will be closed without any changes
